Formula/chapel.rb

```diff
@@ -16,26 +15,30 @@ class Chapel < Formula
   end
 
   depends_on "gmp"
-  depends_on "llvm@11"
+  depends_on "llvm"
```

Comment:
@tzinsky : Not related to the current CI failures, but I think this should probably be "llvm@13" so that if/when the default llvm becomes llvm@14 in the future, the formula won't break?  (since we don't have llvm 14 support yet).

Reply:
(Also, I'm noticing that subsequent commands are hard-coded to llvm@13).

Reply:
The earlier PR failed audit for this.  The error was that the formula should not contain aliases for the most current version.   Hence the change to be llvm versus llvm@13

Reply:
Oh, interesting!

@SMillerDev : Do you have suggestions for avoiding having this break on the day homebrew's default becomes llvm@14?  Since our compiler integrates directly with LLVM, we have to update our compiler sources to become compatible with any new LLVM release (to adjust to the changes to their IR).

Reply:
The best way is to make sure the test will fail when run with LLVM 14 libraries.

Reply:
@Bo98: I suspect that the existing tests will fail when LLVM 14 is the default.  Is the idea then that, when that happens and the failure is triggered, a new formula pinning our formula to llvm@13 would need to be submitted?

If so, would the tripping of that failure be something that would be noticed early and automatically during the LLVM upgrade, or would it require a user to hit it in the field and to let us/someone know?  And would submitting the revised formula be our responsibility or the developers doing the LLVM upgrade?

Thanks for improving our understanding here!

Reply:
When a formula is updated, all dependents are tested. So when LLVM is updated to 14, the test of chapel will be run as a part of CI.

The LLVM 14 upgrade won't be merged with failing CI so if it's obvious that migrating to LLVM 13 will fix the test then that's exactly what will be done before merging.